tcpdf.php

```diff
@@ -2604,9 +2604,9 @@ public function SetMargins($left, $top, $right=-1, $keepmargins=false) {
 	 * @param float $margin The margin.
 	 * @public
 	 * @since 1.4
-	 * @see SetTopMargin(), SetRightMargin(), SetAutoPageBreak(), SetMargins()
+	 * @See setTopMargin(), setRightMargin(), setAutoPageBreak(), setMargins()
 	 */
-	public function SetLeftMargin($margin) {
+	public function setLeftMargin($margin) {
```

Comment:
hum, this is not right, you may not change existing function definitions

Reply:
why not? PHP function calling is case insensitive

Reply:
My point is that the people using the project will not like this change, maybe it will break something
I would recommend you to revert thoes changes because you will risk that the owner of the project does not merge your changes :/

Reply:
I don't see how this could break anything, but I created a new pull request